src/command-handlers/passwords.ts

```diff
@@ -28,7 +27,7 @@ export const runPassword = async (
     }
 
     if (field === 'otp') {
-        foundCredentials = foundCredentials.filter((credential) => credential.otpSecret);
+        foundCredentials = foundCredentials.filter((credential) => credential.otpSecret || credential.otpUrl);
 
         if (foundCredentials.length === 0) {
```

Comment:
Why you don't exit before the if, the rest of the function will have no purpose with no result or ?

Reply:
🤓